### PR TITLE
feat(PropsedSalesLocations): Update the copy and links of the propsed…

### PR DESCRIPTION
**Pull request by kotva006**
… sales locations

#### Summary of changes
Copy update of proposed sales location page

https://app.asana.com/0/555089885850811/1206994520492250

---

#### General checks
* [ ] **Are the changes organized into self-contained commits with descriptive and well-formatted commit messages?** This is a good practice that can facilitate easier reviews.
* [ ] **Testing.** Do the changes include relevant passing updates to tests? This includes updating screenshots. Preferably tests are run locally to verify that there are no test failures created by these changes, before opening a PR.
* [ ] **Tech debt.** Have you checked for tech debt you can address in the area you're working in? This can be a good time to address small issues, or create Asana tickets for larger issues.

#### New UI, or substantial UI changes
* [ ] **Cross-browser compatibility** is less of an issue now that we're no longer supporting IE, but changes still need to work as expected in Safari, Chrome, and Firefox.
* [ ] **Are interactive elements accessible?** This includes at minimum having [relevant keyboard interactions](https://thinkdobecreate.com/articles/4-required-tests-before-shipping-new-features/#keyboard-interaction) and [visible focus](https://thinkdobecreate.com/articles/4-required-tests-before-shipping-new-features/#visible-focus), but can also include verification with screen reader testing.
* [ ] **Other accessibility checks** such as sufficient [color constrast](https://thinkdobecreate.com/articles/4-required-tests-before-shipping-new-features/#color-contrast), or whether the layout holds up at 200% [zoom level](https://thinkdobecreate.com/articles/4-required-tests-before-shipping-new-features/#visible-focus).

#### New endpoints, or non-trivial changes to current endpoints
* [ ] **Have we load-tested** any new pages or internal API endpoints that will receive significant traffic? [See load testing docs](./../load_tests/README.md)
* [ ] **If this change involves routes,** does it work correctly with pertinent "unusual" routes such as the combined Green Line, Silver Line, Foxboro commuter rail, and single-direction bus routes like the 170?

### CODE REVIEW

File: lib/dotcom_web/templates/fare/proposed_sales_locations.html.heex [hunk 1]
```diff
@@ -57,14 +57,14 @@
       <p>
         We're asking the public to help guide our decisions as we roll out the Fare Transformation project over the next several years.
       </p>
-      <%= link "Find out how to get involved", to: "https://faretransformation.mbta.com/get-involved", target: "_blank", class: "c-call-to-action" %>
+      <%= link "Find out how to get involved", to: "https://www.mbta.com/fares/fare-transformation/get-involved", target: "_blank", class: "c-call-to-action" %>
```
Comment (thecristen) [inline lib/dotcom_web/templates/fare/proposed_sales_locations.html.heex:60]: **suggestion**: let's use the helper function to generate this URL, so the hostname can change based on which environment the website's being served from. e.g use `cms_static_page_path/2`.

**Review by thecristen — CHANGES_REQUESTED** [review]

File: lib/dotcom_web/templates/fare/proposed_sales_locations.html.heex (hunk 1 above)
Comment (kotva006) [reply lib/dotcom_web/templates/fare/proposed_sales_locations.html.heex:60]: Do we want to do the same thing with the [retail_sales_locations](https://github.com/mbta/dotcom/blob/main/lib/dotcom_web/templates/fare/retail_sales_locations.html.heex#L42)

Comment (thecristen) [reply lib/dotcom_web/templates/fare/proposed_sales_locations.html.heex:60]: If it needs it, yeah that'd be great!

Comment (kotva006) [reply lib/dotcom_web/templates/fare/proposed_sales_locations.html.heex:60]: It looks like [`cms_static_page_path/2`](https://github.com/mbta/dotcom/blob/main/lib/dotcom_web/views/helpers.ex#L134) currently is just a pass through and has no impact on the generated URL.  

Though changing this to `cms_static_page_page(..., "/fares/fare-trans...")` will achieve the desired outcome as the path is relative which will generate a relative link

**Review by thecristen — APPROVED** [review]

**Pull request merged**